.github/workflows/go_mod_tidy.yml

```diff
@@ -42,6 +42,10 @@ jobs:
       run: |
         inv -e install-tools
         inv -e generate-licenses
+    - name: Update mocks
+      if: ${{ github.event_name == 'workflow_dispatch' || !contains(github.event.pull_request.labels.*.name, 'dependencies-go-tools') }}
```

Comment:
❓ 
Just to be sure, can you elaborate on the purpose of this condition?

Reply:
The goal is to just run this if this is a dependabot bumping a go library (like this one https://github.com/DataDog/datadog-agent/pull/18538), not if it's bumping a python dependency or similar. I just copied the condition from the previous step